test/e2e/specs/fakerp/admin.go

```diff
@@ -27,12 +29,16 @@ import (
 )
 
 var _ = Describe("Openshift on Azure admin e2e tests [AzureClusterReader][Fake]", func() {
+	dataDir, err := shared.FindDirectory(shared.DataDirectory)
```

Comment:
Can we ensure this is used only once during test setup and every test that needs to grab a manifest from the filesystem uses that instead of having to reimplement this logic in every test?